guides/source/upgrading_ruby_on_rails.md

```diff
@@ -845,6 +845,26 @@ want to add this feature it will need to be turned on in an initializer.
 
     config.active_record.belongs_to_required_by_default = true
 
+The configuration is by default global for all your models, but you can
+override it on a per model basis. This should help you migrate all your models to have their
+associations required by default.
+
+    ```ruby
+      class Book < ApplicationRecord
+        # model is not yet ready to have its association required by default
+
+	self.belongs_to_required_by_default = false
```

Comment:
hmm... somehow 954ad6bd92206cec67fdf350a08668a888f8a07c and 3760cff0676e05404175074bcee89bd8d3a3ed82 doesn't fix code highlight...

![image](https://user-images.githubusercontent.com/12642/71068748-b4d96080-21ba-11ea-940f-0721c1e349a1.png)

https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#active-record-belongs-to-required-by-default-option

Reply:
Thanks for trying to fix. I think it's because the "```ruby" isn't correctly indented. Let me fix that